docs/DeveloperGuide.md

```diff
@@ -174,6 +173,73 @@ Classes used by multiple components are in the `seedu.realodex.commons` package.
+For example, missing out on the family size may not be critical for a buyer who is single and is searching for a bachelor pad, but is critical for a family of 7 who needs a large enough house for all 7 of them.
+These fields are also made necessary for both buyers and sellers, even though they may be intuitively be more suitable for either seller or buyer. For example, while the income may not be absolutely necessary for a seller,
+it is important information if the agent would like to make recommendations for future properties for own-stay or investment purposes.
+
+**Optional fields** include: Remark, Birthday
+A real estate agent may not have any remark for a client yet, and wishes to leave Remark empty.
+A real estate agent may also only want to track birthdays of their esteemed clients, and wishes to not include Birthday for the rest. 
+
+**Field Constraints**
+Specific field constraints are described below. They are designed with the users in mind.
+* NAME:
+  * Must be an alphanumeric string
+* PHONE:
+  * Should only contain numbers, and should be at least 3 digits long. This is to account for phone numbers of all types and need not necessarily be from Singapore. We avoid the '+' sign as it is not necessary and can be replaced with the international prefix
+* INCOME:
+  * Income should be an integer and should be at least 0. Decimals provide an unnecessary level of precision.
+* EMAIL:
+  * Emails should be of the format local-part@domain and adhere to the following constraints:
+  1. The local-part should only contain alphanumeric characters and these special characters, excluding the parentheses, (+_.-).
+  2. The local-part may not start or end with any special characters.
+  3. This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods.\
+     The domain name must:
+    * end with a domain label at least 2 characters long
+    * have each domain label start and end with alphanumeric characters
+    * have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
+* ADDRESS:
+  * String
+* FAMILY:
+  * Should be an integer greater than 1.
+* TAG:
+  * Should be restricted to case-insensitive "buyer" or "seller" using enums.
+* HOUSINGTYPE: housing type a buyer wants or housing type a seller is selling
+  * Should be restricted to "HDB", "CONDOMINIUM", "LANDED PROPERTY", "GOOD CLASS BUNGALOW" (case-insensitive) using enums. 
+  * Only one housing type is allowed.
+* REMARK:
+  * Represented as a String
+  * If remark is not specified, an empty string is used for representation 
+* BIRTHDAY:
+  * Implemented as an Optional Date, making use of Java SimpleDateFormat and Calendar classes for input and output validation.
+
+The sequence diagram below illustrates the process of adding a person into Realodex.
+<puml src="diagrams/add/AddCommandSequenceDiagram.puml" width="800" />
```

Comment:
Change width to 1000


After changing argument it should look like 

![Uploading image.png…]()

Reply:
Thanks. I made the necessary changes.